pandas/tests/plotting/test_misc.py

```diff
@@ -277,6 +274,32 @@ def test_radviz(self):
         handles, labels = ax.get_legend_handles_labels()
         self._check_colors(handles, facecolors=colors)
 
+    @slow
+    def test_subplot_titles(self):
+        df = self.iris.drop('Name', axis=1).head()
+        # Use the column names as the subplot titles
+        title = list(df.columns)
+
+        # Case len(title) == len(df)
+        plot = df.plot(subplots=True, title=title)
+        self.assertEqual([p.title._text for p in plot], title)
```

Comment:
Sorry, I missed this earlier, but can you use `p.get_title()` instead of the private matplotlib `._text` method. Same thing down on line 300, `ax.get_title()`.